runtime/compiler/p/codegen/J9PPCWatchedStaticFieldSnippet.hpp

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef J9PPCWATCHEDSTATICFIELDSNIPPET_INCL
+#define J9PPCWATCHEDSTATICFIELDSNIPPET_INCL
+
+#include "codegen/J9WatchedStaticFieldSnippet.hpp"
+
+#include "env/VMJ9.h"
+#include "infra/Annotations.hpp"
+#include "p/codegen/PPCInstruction.hpp"
+
+#include "codegen/Instruction.hpp" //TR:Instruction
+#include "env/TRMemory.hpp" 
+
+namespace TR  {
+
+class J9PPCWatchedStaticFieldSnippet: public TR::J9WatchedStaticFieldSnippet
+    {
```

Comment:
It looks like the new files don't follow coding conventions. There are too many spaces before the braces begin. It should be (3 spaces):
```
class J9PPCWatchedStaticFieldSnippet: public TR::J9WatchedStaticFieldSnippet
   {
```

Similar concept can be applied to the new `if/else` statements and function definitions.